crates/bevy_ecs/src/archetype.rs

```diff
@@ -116,6 +123,9 @@ pub(crate) struct ArchetypeComponentInfo {
     pub(crate) archetype_component_id: ArchetypeComponentId,
 }
 
+/// A unique set of components stored within the [`World`](crate::world::World)
```

Comment:
With sparse set components, these can contain many sets of component, although each set is uniquely assigned to a single archetype.